javascripts/govuk/analytics/google-analytics-universal-tracker.js

```diff
@@ -33,24 +34,25 @@
 
   // https://developers.google.com/analytics/devguides/collection/analyticsjs/pages
   GoogleAnalyticsUniversalTracker.prototype.trackPageview = function(path, title, options) {
-    var options = options || {};
+    var options;
```

Comment:
Don’t think we need this line right? `options` is coming in as an argument and if it’s not supplied `if (typeof options === "object")` will just fail as `options` will equal `'undefined'`.